core/trino-main/src/main/java/io/trino/operator/scalar/MathFunctions.java

```diff
@@ -702,6 +703,32 @@ public static double betaCdf(
         return distribution.cumulativeProbability(value);
     }
 
+    @Description("Students' t-distribution cdf given the x and degrees of freedom")
+    @ScalarFunction
+    @SqlType(StandardTypes.DOUBLE)
+    public static double tCdf(
+            @SqlType(StandardTypes.DOUBLE) double x,
+            @SqlType(StandardTypes.DOUBLE) double degreesOfFreedom)
+    {
+        if (degreesOfFreedom < 1) {
+            throw new TrinoException(INVALID_FUNCTION_ARGUMENT, "degrees of freedom must be greater than or equal to 1");
+        }
+        return new TDistribution(degreesOfFreedom).cumulativeProbability(x);
+    }
+
+    @Description("Students' t-distribution given the x and degrees of freedom")
+    @ScalarFunction(alias = "t_pdf")
```

Comment:
Let's just got with `t_pdf`. "distribution" is a broader concept, and pdf/cdf describe certain properties of a distribution. In particular, pdf describes density of probability of a random variable at a given point for the distribution.